src/emptyPackageProject/Private.SourceBuild.ReferencePackages.csproj

```
@@ -11,10 +11,15 @@
     <RepositoryCommit>0000</RepositoryCommit>
     <RepositoryCommit Condition="'$(ManifestCommit)' != ''">$(ManifestCommit)</RepositoryCommit>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageIcon>icon.png</PackageIcon>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <IsPackable>true</IsPackable>
     <IsShipping>false</IsShipping>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Include="icon.png" Pack="true" PackagePath=""/>
```

(Nit) The usual xml style I see always includes a space before `/>` for clearer punctuation:

```suggestion
    <None Include="icon.png" Pack="true" PackagePath="" />
```